[2022-11-02T18:20:47.314+0700] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: postgres_dag.create_table manual__2022-11-02T11:20:45.045806+00:00 [queued]>
[2022-11-02T18:20:47.319+0700] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: postgres_dag.create_table manual__2022-11-02T11:20:45.045806+00:00 [queued]>
[2022-11-02T18:20:47.319+0700] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-02T18:20:47.319+0700] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2022-11-02T18:20:47.319+0700] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-02T18:20:47.329+0700] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2022-11-02 11:20:45.045806+00:00
[2022-11-02T18:20:47.332+0700] {standard_task_runner.py:55} INFO - Started process 98603 to run task
[2022-11-02T18:20:47.334+0700] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'postgres_dag', 'create_table', 'manual__2022-11-02T11:20:45.045806+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres.py', '--cfg-path', '/tmp/tmp8dvyr9ii']
[2022-11-02T18:20:47.335+0700] {standard_task_runner.py:83} INFO - Job 50: Subtask create_table
[2022-11-02T18:20:47.377+0700] {task_command.py:376} INFO - Running <TaskInstance: postgres_dag.create_table manual__2022-11-02T11:20:45.045806+00:00 [running]> on host ginraca-VirtualBox
[2022-11-02T18:20:47.411+0700] {taskinstance.py:1590} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=postgres_dag
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-11-02T11:20:45.045806+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-02T11:20:45.045806+00:00
[2022-11-02T18:20:47.414+0700] {base.py:71} INFO - Using connection ID 'postgres_localhost' for task execution.
[2022-11-02T18:20:47.429+0700] {sql.py:315} INFO - Running statement: 
            create table if not exists dag_coba(
                dt date,
                dag_id varchar,
                primary key (dt, dag_id)
            )
        , parameters: None
[2022-11-02T18:20:47.430+0700] {postgres.py:96} INFO - NOTICE:  relation "dag_coba" already exists, skipping

[2022-11-02T18:20:47.434+0700] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=postgres_dag, task_id=create_table, execution_date=20221102T112045, start_date=20221102T112047, end_date=20221102T112047
[2022-11-02T18:20:47.470+0700] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-02T18:20:47.479+0700] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
